pages/Addons.md

```diff
@@ -63,6 +63,7 @@ AlchimiaVitae | Apeiros-46B | <details><summary>expand</summary>An addon that ad
 SlimeTinker | Sefiraat | <details><summary>expand</summary>SlimeTinker is a Slimefun4 addon that tries its best to recreate the amazing [Tinker's Construct Mod](https://www.curseforge.com/minecraft/mc-mods/tinkers-construct) in Vanilla Minecraft. Melt metals, alloy them together, cast metals out into blocks, nuggets, tool parts and more. Build custom tools with properties dependant on the metals used. Tools do not break but instead they can be repaired with kits. Swap out parts as wanted and add special modifiers.</details> | [Wiki](https://github.com/Sefiraat/SlimeTinker/wiki) | [GitHub](https://github.com/Sefiraat/slimeTinker) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/SlimeTinker/master/)
 PotionExpansion | EpicPlayerA10 | <details><summary>expand</summary>An addon which adds new potions that give you legal xray!</details> | [Readme](https://github.com/EpicPlayerA10/PotionExpansion/blob/master/README.md) | [GitHub](https://github.com/EpicPlayerA10/PotionExpansion) | [Builds](https://thebusybiscuit.github.io/builds/EpicPlayerA10/PotionExpansion/master/)
 FlowerPower | NCBPFluffyBear | <details><summary>expand</summary>FlowerPower is a magic addon centered around experience and flowers. The base of each recipe requires various flowers, and different items are crafted along the way to make mid to endgame items. A majority of these items function by consuming experience in exchange for a certain effect or buff.</details> | [Readme](https://github.com/NCBPFluffyBear/FlowerPower/blob/master/README.md) | [GitHub](https://github.com/NCBPFluffyBear/FlowerPower/) | [Builds](https://thebusybiscuit.github.io/builds/NCBPFluffyBear/FlowerPower/master/)
+Galactifun | Seggan, Mooy1, GallowsDove, ProfElements | <details><summary>expand</summary>Galactifun is an addon inteded to replace the apparently abandoned ClayTech. It adds multiple planets, rockets, space stations, and more!</details> | [Readme](https://github.com/Slimefun-Addon-Community/Galactifun/blob/master/README.md) | [GitHub](https://github.com/Slimefun-Addon-Community/Galactifun/) | [Builds](https://thebusybiscuit.github.io/builds/Slimefun-Addon-Community/Galactifun/master/)
```

Comment:
```suggestion
Galactifun | Seggan, Mooy1, GallowsDove, ProfElements | <details><summary>expand</summary>Galactifun is an addon inspired by ClayTech. It adds multiple planets, rockets, space stations, and more!</details> | [Readme](https://github.com/Slimefun-Addon-Community/Galactifun/blob/master/README.md) | [GitHub](https://github.com/Slimefun-Addon-Community/Galactifun/) | [Builds](https://thebusybiscuit.github.io/builds/Slimefun-Addon-Community/Galactifun/master/)
```

ClayTech is not "apparently abandoned". It had updated forks and the last commit was 2 weeks ago. This has been stated time and time again and this is just misleading.